gui/builtinItemStatsViews/itemProperties.py

```diff
@@ -95,5 +95,5 @@ def PopulateList(self):
 
         self.paramList.SortItems(lambda id1, id2: (idNameMap[id1] > idNameMap[id2]) - (idNameMap[id1] < idNameMap[id2]))
         self.paramList.RefreshRows()
-        self.totalAttrsLabel.SetLabel("%d attributes. " % idCount)
+        self.totalAttrsLabel.SetLabel(_t("%d attribute.", "%d attributes.", idCount) % idCount)
```

Comment:
Oh wow, I did not realize wx.GetTranslation had another argument for plural strings. Nice find! I'm sure there's other places in the codebase where using this would be helpful